docs-src/typescript/chapter-project-setup/install-cli.md

```diff
@@ -0,0 +1,82 @@
+---
+id: install-cli
+title: Install the Temporal CLI
+description: Download and install the Temporal CLI for Mac, Linux, or Windows.
+sidebar_label: Install CLI
+tags:
+  - temporal cli
+  - dev guide
+---
+
+**How to download and install the Temporal CLI**
+
+The Temporal CLI is available on MacOS, Windows, and Linux.
+
+### MacOS
+
+**How to install the Temporal CLI on Mac OS**
+
+Choose one of the following install methods to install the Temporal CLI on MacOS:
+
+**Install the Temporal CLI with Homebrew**
+
+```bash
+brew install temporal
+```
+
+**Install the Temporal CLI with cURL**
+
+```bash
+curl -sSf https://temporal.download/cli.sh | sh
+```
+
+**Install the Temporal CLI from CDN**
+
+1. Select the platform and architecture needed.
+
+- Download for Darwin amd64: https://temporal.download/cli/archive/latest?platform=darwin&arch=amd64
+- Download for Darwin arm64: https://temporal.download/cli/archive/latest?platform=darwin&arch=arm64
+
+2. Extract the downloaded archive.
+
+3. Add the `temporal` binary to your PATH.
+
+### Linux
+
+**How to install the Temporal CLI on Linux**
+
+Choose one of the following install methods to install the Temporal CLI on Linux:
+
+**Install the Temporal CLI with cURL**
+
+```bash
+curl -sSf https://temporal.download/cli.sh | sh
+```
+
+**Install the Temporal CLI from CDN**
```

Comment:
Line 56-65 are the same as lines 33-42 (you have the same Install the Temporal CLI from CDN section)